pom.xml

```diff
@@ -673,6 +674,11 @@
         <artifactId>junit</artifactId>
         <version>${junit.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.tngtech.junit.dataprovider</groupId>
```

Comment:
do we really need to add this new dependency ?
in junit4 there is support for parametrised tests
https://github.com/junit-team/junit4/wiki/Parameterized-tests
https://www.testwithspring.com/lesson/writing-parameterized-tests-with-junit-4/

Reply:
Already move `OpenLedgerTest` to a new test class `BookKeeperBuildersOpenLedgerTest` and used feature `junit-parametrised`